docs/wiki/end-user-guide/preparing-your-modpack.md

```diff
@@ -0,0 +1,94 @@
+
+::: info Note {id="note"}
+**_Celeritas_** and **_Nothirium_** do decrease ram usage and footprint, While **_Optifine_** may increase ram usage and will lengthen load times. 
+:::
+### Stress Test Results
+In a simulated heavy-base scenario, we benchmarked the aforementioned mods against each other, and therefore rank their performance according to which gpu is used: <br/>
+- **NVIDIA GPUs** → `Nothirium > Optifine > Celeritas`
+- **AMD GPUs** → `Celeritas > Optifine > Nothirium`
+- **Intel Integrated Graphics** → `Celeritas | Optifine` <br/> <br/>
+:warning: Performance varies depending on your hardware and drivers. Test all three mods to see which provides the best results for your system.
+
+## Superceded mods 
+::: info Note {id="note"}
+These mods are not necessarily incompatible with Cleanroom loader, but have been superceded by newer forks or do not present any benefit, so we recommend you use to follow the actions we recommend
+:::
+
+| Mod                         | Reason                                                                                                 | Recommended Action                                                                                                               |
+|-----------------------------|--------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------|
+| **BetterFPS**               | Breaks core functionality.                                                                             | :no_entry_sign: Remove                                                                                                           |
+| **Betterer Biome Blender**  | Minor fixes for an already useless mod.                                                                | :no_entry_sign: Remove                                                                                                           |
+| **Clumps**                  | Outdated                                                                                               | :white_check_mark: Use **[Fixeroo](https://www.curseforge.com/minecraft/mc-mods/xp-orb-clump)**                                  |
+| **FastFurnace**             | Merged and Rewritten in CensoredASM                                                                    | :white_check_mark: Use **[CensoredASM](https://www.curseforge.com/minecraft/mc-mods/lolasm)**                                    |
+| **FermiumASM**              | CensoredASM fork, disables fixes with Dynamic Trees, depends on FermiumBooter.                         | :white_check_mark: Use **[CensoredASM](https://www.curseforge.com/minecraft/mc-mods/lolasm)**                                    |
```

Comment:
whats the source on disabling fixing with Dynamic Trees? doesnt seem accurate from the source code.